test/module/irohad/ordering/ordering_service_test.cpp

```diff
@@ -296,3 +294,35 @@ TEST_F(OrderingServiceTest, GenerateProposalDestructor) {
   }
   EXPECT_CALL(*fake_transport, publishProposalProxy(_, _)).Times(0);
 }
+
+/**
+ * Check that batches are processed by ordering service
+ * @given ordering service up and running
+ * @when feeding the ordering service two batches, such that number of
+ * transactions in both summed is greater than maximum number of transactions
+ * inside proposal
+ * @then proposal will still contain number of transactions, equal to sum of the
+ * batches
+ */
+TEST_F(OrderingServiceTest, BatchesProceed) {
+  const auto max_proposal = 12;
+
+  auto batch_one = framework::batch::createValidBatch(10);
+  auto batch_two = framework::batch::createValidBatch(5);
+
+  EXPECT_CALL(*fake_persistent_state, saveProposalHeight(_))
+      .Times(1)
+      .WillRepeatedly(Return(true));
+  EXPECT_CALL(*fake_persistent_state, loadProposalHeight())
+      .Times(1)
+      .WillOnce(Return(boost::optional<size_t>(15)));
```

Comment:
please make 15 a constant. Even better is to replace with expession (batch1.size + batch2.size)